lib/combo.coffee

```diff
@@ -100,6 +103,9 @@
     constructor: (wrapper, options = {}) ->
       @[key] = value for own key, value of options when value?
 
+      if @forceSelectionFromList and @showUnmatchedRawValue
+        throw new Error('forceSelectionFromList and showUnmatchedRawValue is not cooperable')

```

Comment:
invalid configuration, forceSelectionFromList and showUnmatchedRawValue cannot both be true